caom2-search-server/src/main/resources/META-INF/resources/images/wmms.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should be in AdvancedSearch only.  It does not apply to this open source version.